Arrowgene.Ddon.Database/Files/Database/Script/schema_sqlite.sql

```diff
@@ -428,3 +429,17 @@ CREATE TABLE IF NOT EXISTS ddon_connection
     CONSTRAINT uq_connection_server_id_account_id UNIQUE (server_id, account_id),
     CONSTRAINT fk_connection_token_account_id FOREIGN KEY ("account_id") REFERENCES account ("id")
 );
+
+CREATE TABLE IF NOT EXISTS "ddon_contact_list" (
+    "id"                        INTEGER UNIQUE,
+    "requester_character_id"	INTEGER NOT NULL,
+    "requested_character_id"	INTEGER NOT NULL,
+    "status"                    SMALLINT NOT NULL,
+    "type"                      SMALLINT NOT NULL,
+    "requester_favorite"	    BOOLEAN NOT NULL,
+    "requested_favorite"	    BOOLEAN NOT NULL,
+    FOREIGN KEY("requester_character_id") REFERENCES "ddon_character"("character_id"),
+    FOREIGN KEY("requested_character_id") REFERENCES "ddon_character"("character_id"),
+    PRIMARY KEY("id" AUTOINCREMENT),
+    UNIQUE("requester_character_id","requested_character_id")
```

Comment:
Why not use these two fields as the primary key? It's fine either way, just a question

Reply:
just because some of the packets have a unique friend id field instead of a character id, so i used this id as the unique friend id. I can probably just use the character id if you want

Reply:
If the packets do have that, then it's better to also have it in our code too, thanks.